content/nginxaas-azure/module-changelog.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Module Changelog"
+weight: 950
+toc: true
+url: /nginxaas/azure/module-changelog/
+---
+
+Learn about the modules supported by the latest versions of F5 NGINX as a Service for Azure.
+
+
+## June 10, 2025 
+
+### Preview
+
+ {{<bootstrap-table "table table-bordered table-striped table-responsive table-sm">}}
+
+| Name                                     | Version                  | Description                                                            |
+|------------------------------------------|--------------------------|------------------------------------------------------------------------|
+| app-protect-module-plus                  | 33+5.264.0-1~jammy         | App-Protect package for Nginx Plus, includes all of the default files and examples. NGINX App Protect provides web application firewall (WAF) security protection for your web applications, including OWASP Top 10 attacks. |
+|  app-protect-plugin                   | 6.9.0-1~jammy          | NGINX App Protect plugin |
+| nginx-agent                              | 1.19.15-1795423089       | NGINX Agent - Instance Management for NGINX; quick, fast, and easy.    |
+| nginx-plus                               | 33-3~jammy               | NGINX Plus, provided by Nginx, Inc.                                    |
+| nginx-plus-module-appprotect             | 33+5.264.0-1~jammy       | NGINX Plus app protect dynamic module version 5.264.0                  |
+| nginx-plus-module-geoip2                 | 33+3.4-1~jammy           | NGINX Plus 3rd-party GeoIP2 dynamic modules                            |
+| nginx-plus-module-headers-more           | 33+0.37-1~jammy          | NGINX Plus 3rd-party headers-more dynamic module                       |
+| nginx-plus-module-image-filter           | 33-1~jammy               | NGINX Plus image filter dynamic module                                 |
+| nginx-plus-module-lua                    | 33+0.10.27-1~jammy       | NGINX Plus 3rd-party Lua dynamic modules                               |
+| nginx-plus-module-ndk                    | 33+0.3.3-1~jammy         | NGINX Plus 3rd-party NDK dynamic module                                |
+| nginx-plus-module-njs                    | 33+0.8.9-1~jammy         | NGINX Plus njs dynamic modules                                         |
+| nginx-plus-module-otel                   | 33+0.1.0-1~jammy         | NGINX Plus OpenTelemetry dynamic module                                |
+| nginx-plus-module-xslt                   | 33-1~jammy               | NGINX Plus xslt dynamic module                                         |
+
+{{</bootstrap-table>}}
+
+### Stable
```

Comment:
Given that >95% of our deployments are using `Stable`, can we put this info ahead of the `Preview` section so folks don't have to scroll down?